recipes/hmat-oss/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+patch -p1 -i ${RECIPE_DIR}/7d14b6451d3f63a949fee9c09d349aec6edd193e.patch
+patch -p1 -i ${RECIPE_DIR}/2bea755a2139907d331bfa0229154b6dab7a61c1.patch
+patch -p1 -i ${RECIPE_DIR}/aad9de409bc2e0a8a50fed1169df0a06a252f399.patch
+
+if test `uname` = "Darwin"
```

Comment:
@jakirkham no, that's bash-specific syntax the script explicitely calls sh: #!/bin/sh